charts/alfresco-adf-app/templates/deployment.yaml

```diff
@@ -38,16 +38,12 @@ spec:
           {{- end }}
           ports:
             - name: http
-              containerPort: {{ .Values.service.port }}
+              containerPort: {{ .Values.service.targetPort }}
```

Comment:
I am confused, this should be the same port as the port exposed by the image itself - which should be 80 - how the application can be reachable if this one is now 8080?